### ISSUE TITLE
Add support for Sonix sn8 MCU architecture

### ISSUE THREAD

**Issue by Marisa-Chan** [radare/radare2#9427]
[Datasheets](http://www.sonix.com.tw/list-en-2515)
[Simple python disassembler](https://github.com/Marisa-Chan/sn8dis)

**Comment by montekki** [comment]
@Marisa-Chan you have the firmware dumps?

**Comment by Marisa-Chan** [comment]
Yes, https://drive.google.com/open?id=1BI4XG8ZoGlCNw0YoH4O2D4tT29BvfPNJ dump.
MCU is sn8f2253b

**Comment by Marisa-Chan** [comment]
https://drive.google.com/open?id=1gAm2dkZO9I5muxRYyuTQsqG4gVL3T7T0
sn8f2251b

**Comment by Marisa-Chan** [comment]
Any another binary can be compiled in official free C studio  : http://www.sonix.com.tw/list-en-2747

**Comment by ret2libc** [comment]
This issue has been moved from radareorg/radare2 to radareorg/ideas as we are trying to clean our backlog and this issue has probably been created a long while ago. This is an effort to help contributors understand what are the actionable items they can work on, prioritize issues better and help users find active/duplicated issues more easily. If this is not an enhancement/improvement/general idea but a bug, feel free to ask for re-transfer to main repo. Thanks for your understanding and contribution with this issue.
